src/browser/base/content/zen-styles/zen-popup.css

```diff
@@ -196,6 +196,36 @@ panel {
   min-height: calc((var(--arrowpanel-menuitem-padding-block) + 4px) * 2 + 16px);
 }
 
+/* Workspace icon picker styles */
+#PanelUI-zen-workspaces-icon-picker-wrapper {
```

Comment:
Please move this into the workspaces css files

Reply:
lol i wasnt sure cuz it was a popup but it also had to do with workspaces mb.